### DISCUSSION TITLE
Shortcut for Video/audio to text

### DISCUSSION THREAD

**Discussion by devkatalist** [SubtitleEdit/subtitleedit#5892] [Ideas]
Hello. As always, thank you so much for SE. You are amazing!
So, just like the title says, is there a way to add a shortcut for the video to text option? That would be helpful. Thanks!

**Comment by niksedk** [comment]
Sure, in latest beta :)
https://github.com/SubtitleEdit/subtitleedit/releases/download/3.6.5/SubtitleEditBeta.zip
